src/StackExchange.Redis/ConnectionMultiplexer.cs

```diff
@@ -465,7 +467,7 @@ internal void MakeMaster(ServerEndPoint server, ReplicationChangeOptions options
             // We want everyone possible to pick it up.
             // We broadcast before *and after* the change to remote members, so that they don't go without detecting a change happened.
             // This eliminates the race of pub/sub *then* re-slaving happening, since a method both precedes and follows.
-            void Broadcast(ReadOnlySpan<ServerEndPoint> serverNodes)
+            async Task BroadcastAsync(ServerEndPoint[] serverNodes)
```

Comment:
minor nit: if the use of arrays is to avoid span+async (which is illegal) - perhaps consider `ReadOnlyMemory<T>` instead of `T[]`

Reply:
Updating here: chatted on this, it won't actually do an array copy and the bigger change touches a lot more, so we'll go with current and see about `ReadOnlyMemory<ServerEndPoint>` in a dedicated PR (affects many `foreach` loops and such, creating chunk of changes).